release_notes/topics/product/7_4_final.adoc

```diff
@@ -0,0 +1,180 @@
+
+Users can manage their WebAuthn credential only in the New Account Console as described in the next section.
+
+== New Account Console (Preview)
+
+The User Account Service is being significantly improved as a new Account Console on a Technology Preview basis. The existing User Account Service is still supported.
+
+To experiment in using this console,
+
+. Start the RH-SSO server with the system properties to enable new Account Console and the new Account REST API:
+
++
+standalone -Dkeycloak.profile.feature.account_api=enabled -Keycloak.profile.feature.account2=enabled
++
+. Log into the Admin Console.
+. Select Realm Settings, Themes.
+. Change the Account Theme to *keycloak-preview*.
+
+When you go to Manage Account, you will see the new Account Console.  Here is a sample screen:
+
+image:{project_images}/device-activity.png[]
+
+This new console is based on link:https://reactjs.org/[React] and link:https://www.patternfly.org/v4/[PatternFly 4].  It allows you to use link:https://www.patternfly.org/v4/documentation/overview/global-css-variables[PatternFly CSS variables] for easy styling.  It also allows you to remove pages and add your own pages.  Full documentation will be provided at a future release.
+
+== New default hostname provider
+
+This new default hostname provider adds the following improvements:
+
+* No need to change provider to a set fixed base URL
+
+* Support of a different base URL for frontend and backend requests
+
+* Support for changing context-path when RH-SSO is exposed on a different context-path through a reverse proxy
+
+== Additional improvements
+
+RH-SSO includes additional minor features at this release. Most of those features improve support for advanced OpenID Connect/OAuth2 concepts and algorithms, which is related to support of Financial-grade API (FAPI). RH-SSO does not yet fully support FAPI, but the following changes move in that direction.
+
+* MP-JWT Client Scope, making it easy to issue tokens following the Eclipse MicroProfile specification.
+
+* More algorithms supported for client authentication with signed client secret JWT. Namely HS384 and HS512 algorithms were added to an already existing HS256 algorithm.
+
+* Client authentication for OIDC identity brokering with signed JWT or basic authentication.  All client authentication methods in the OIDC specification are supported.
+
+* Identity brokering changes to make it easier to disable auto-creation of RH-SSO users during the first login of a particular user with an identity provider. See the link:{adminguide_link}[{adminguide_name}] for more details.
+
+* Signed and encrypted ID token support according to the Json Web Encryption (JWE) specification.
+
+* Support for additional signing algorithms for client authentication with signed JWT. All supported algorithms are RS256, RS384, RS512, PS256, PS384, PS512, ES256, ES384 and ES512
```

Comment:
Could be first sentence changed to either "Support for additional signing algorithms for client authentication with JWT signed by private key." or "Support for additional signing algorithms for client authentication with private key signed JWT" ? It will be nice to mention private key here. Sorry to not mention it before when you worked on the document.

Reply:
Okay, I used the second suggestion. Thanks, Marek.